docs/API.md

```diff
@@ -604,10 +607,14 @@ immediately upon experiencing a "gap" in sequences.
 | `SRTO_MAXBW`          | 1.0.5 | pre     | `int64_t` | bytes/sec | -1       | -1     |
 
 - **[GET or SET]** - Maximum send bandwidth.
-- `-1`: infinite (CSRTCC limit is 30mbps)
-- `= 0`: relative to input rate (SRT 1.0.5 addition, see `SRTO_INPUTBW`) 
-- `>0`: absolute limit 
-- *SRT recommended value: 0 (relative)*
+- `-1`: infinite (the limit in Live Mode is 1Gbps)
+- `0`: relative to input rate (see `SRTO_INPUTBW`) 
+- `>0`: absolute limit in B/s
+
+- *NOTE: This option has a default value -1, but for cases when the stream rate
```

Comment:
> This option has a default value -1,

Maybe:
_This option has a default value of -1,_

Cange ", but" with a new sentence.